packages/stark-ui/src/modules/input-mask-directives/directives/text-mask.constant.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Util class containing some constants with predefined masks for common mask patterns to be used with the {@link StarkTextMaskDirective}.
+ */
+export class StarkTextMasks {
+	/**
+	 * Mask for credit card numbers: "dddd-dddd-dddd-dddd"
+	 */
+	public static CREDITCARD_NUMBER = "0000-0000-0000-0000";
+
+	/**
+	 * Mask for Belgian Structured Communication numbers: "+++ddd/dddd/ddddd/+++"
```

Comment:
Could you fix the typo (the final `/`) in the comment ?
It should be `Mask for Belgian Structured Communication numbers: "+++ddd/dddd/ddddd+++"`